mpv.el

```diff
@@ -323,6 +328,28 @@ This can be used with the `org-open-at-point-functions' hook."
   (interactive "p")
   (mpv-speed-increase (- steps)))
 
+;;;###autoload
+(defun mpv-volume-set (factor)
+  "Set playback volume to FACTOR."
+  (interactive "nFactor: ")
+  (mpv--enqueue `("set" "volume" ,(abs factor)) #'ignore))
+
+;;;###autoload
+(defun mpv-volume-increase (steps)
+  "Increase playback volume by STEPS factors of `mpv-volume-step'."
+  (interactive "p")
+  (let ((factor (* (abs steps)
+                   (if (> steps 0)
+                       mpv-volume-step
+                     (/ 1 mpv-volume-step)))))
```

Comment:
Oh, this points to an existing bug in `mpv-speed-increase`. :slightly_smiling_face: Shouldn't it be the following?
```suggestion
  (let ((factor (if (>= steps 0)
                    (* steps mpv-volume-step)
                  (/ 1 (* (- steps) mpv-volume-step)))))
```
If you agree, could you fix it in both places?